lms/djangoapps/teams/static/teams/js/collections/topic.js

```diff
@@ -1,4 +1,4 @@
-;(function (define) {
+(function(define) {
```

Comment:
It's a pattern that arose from when minifiers/module systems were worse than they are today - starting your file with a semicolon could "save" you from your file being concatenated after a file that didn't end with one. It's not a pattern we need to use any more, I think.

cc @andy-armstrong